### ISSUE TITLE
Navigation toggle issues

### ISSUE THREAD

**Issue by jimlongo56** [tmcw/docbox#76]
First of all thanks so much for this, it's helping us a lot to get nice looking maintainable documentation for our users. So please excuse some of the issues i'm going to nitpick in a few issues to come. 

I saw a couple earlier issues about the navigation, I was running into this as well. So I went back to your example to make some sense of what is happening.

What caused this for me was any duplication in names between the level 1 headings and any others.

You could see this using the example files.
If you were to duplicate example.md call it example2.md
Change all occurences of `Wobble` to `Tribble` in example2.md

Then build this.
```
module.exports =
    '# Introduction\n' +
    fs.readFileSync('./content/introduction.md', 'utf8') + '\n' +
    '# Example\n' +
    fs.readFileSync('./content/example.md', 'utf8') + '\n' +
    '# Example2\n' +
    fs.readFileSync('./content/example2.md', 'utf8');
```

You'll have one long group of level 3 headings under Example > Wobble.

First thing is you need to put a double carriage return in line 5. _Not sure why that's happening because that particular bug doesn't happen in most cases for me._

Then you'll have a situation where you'll have both level 2 sections partially opened no matter which you click on.

Solution to that is to change all occurrences of `wibble` with `dribble` in example2.md

Bottom line is you need to make sure that there are absolutely no duplication of headings; Level 1, level 2 and level 3 headings must all be unique. If you know that you can correct for it, but in a long document it can be a little challenging.

**Comment by tmcw** [comment]
Have you tried running the test suite? The [tests](https://github.com/tmcw/docbox#testing-driven) include assertions that headlines are unique, and that's been an intended attribute - otherwise linking to a heading is no longer reliable - if you have two headings named `Wobble`, then the only thing that we can differentiate them by is their order, like `Wobble-1` and `Wobble-2`, and if you reverse them for any editorial reason, the links will be incorrect. That's why the thesis has been that headings should be unique.

**Comment by jimlongo56** [comment]
Thanks. I neglected that, so I just ran it on my actual content.

Can you decipher this? 
```
Jims-Mac-mini:docbox jimlongo$ npm test

> docbox@1.0.9 test /Users/jimlongo/docbox
> npm run test-unit && eslint src && npm run prerender -- /dev/null && rm -rf lib


> docbox@1.0.9 test-unit /Users/jimlongo/docbox
> mocha --compilers js:babel-register test

fs.js:675
  return binding.read(fd, buffer, offset, length, position);
                 ^

Error: EISDIR: illegal operation on a directory, read
    at Object.fs.readSync (fs.js:675:18)
    at tryReadSync (fs.js:540:20)
    at Object.fs.readFileSync (fs.js:583:19)
    at Suite.<anonymous> (/Users/jimlongo/docbox/test/content.js:60:24)
    at Object.create (/Users/jimlongo/docbox/node_modules/mocha/lib/interfaces/common.js:112:19)
    at context.describe.context.context (/Users/jimlongo/docbox/node_modules/mocha/lib/interfaces/bdd.js:44:27)
    at /Users/jimlongo/docbox/test/content.js:59:5
    at Array.forEach (<anonymous>)
    at Suite.<anonymous> (/Users/jimlongo/docbox/test/content.js:58:31)
    at Object.create (/Users/jimlongo/docbox/node_modules/mocha/lib/interfaces/common.js:112:19)
    at context.describe.context.context (/Users/jimlongo/docbox/node_modules/mocha/lib/interfaces/bdd.js:44:27)
    at Object.<anonymous> (/Users/jimlongo/docbox/test/content.js:57:1)
    at Module._compile (module.js:643:30)
    at loader (/Users/jimlongo/docbox/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/jimlongo/docbox/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at /Users/jimlongo/docbox/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/Users/jimlongo/docbox/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/Users/jimlongo/docbox/node_modules/mocha/lib/mocha.js:536:10)
    at Object.<anonymous> (/Users/jimlongo/docbox/node_modules/mocha/bin/_mocha:582:18)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! docbox@1.0.9 test-unit: `mocha --compilers js:babel-register test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the docbox@1.0.9 test-unit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jimlongo/.npm/_logs/2018-02-20T17_41_29_544Z-debug.log
npm ERR! Test failed.  See above for more details.
```

**Comment by tmcw** [comment]
According to the line:

```
at Suite.<anonymous> (/Users/jimlongo/docbox/test/content.js:60:24)
```

Which corresponds to [this line](https://github.com/tmcw/docbox/blob/d8aacd79acdcdece79e84be280b827a3a8760b57/test/content.js#L60), my best guess would be... do you have nested folders in `content`?
